src/components/AlbumName.jsx

```diff
@@ -0,0 +1,17 @@
+import { PropTypes } from "prop-types";
+
+export const AlbumName = ({ albumName, uri }) => {
+  return (
+    <div>
+      <div>Album name: {albumName}</div>
+      <div>
+        <a href={uri}>ALBUM LINK</a>
```

Comment:
we can wrap line 6 in the `<a> tag` instead of doing it this way.

Reply:
Nice! Changed in the most recent push